ARCs/arc-0034.md

```diff
@@ -0,0 +1,45 @@
+---
+arc: 34
+title: Governance proposal process
+description: Criteria for the creation of a governance proposal.
+author: Stéphane Barroso (@SudoWeezy), Adriana Belotti, Massimo Morini, Michel Treccani, John Woods, Shai Halevi
+discussions-to: https://github.com/algorandfoundation/ARCs/issues/
+status: Draft
+type: Meta
+created: 2022-11-22
+---
+ 
+## Abstract
```

Comment:
As I understand it now, the purpose of ARC 34 is to stand as a structural template for non-xGovs (and possibly xGovs depending on how ARC 33 is structured) to submit proposals to be filtered and up/down voted through various vertical channels sorted to correspond with an Algorand Ventures vertical team structure before moving on to a General Governance vote based on a DAO xGov prioritized workflow.